src/misc/serial/toitserial.ts

```diff
@@ -0,0 +1,202 @@
+import "../../service/proto/uuid";
```

Comment:
this file only contains things that works in together with the console, remove the file